x-pack/plugins/maps/public/components/layer_addpanel/source_select/_source_select.scss

```diff
@@ -8,3 +8,7 @@
     padding-top: 2px !important;
   }
 }
+
+.mapLayerImportpanel__card {
+  border: 1px solid rgb(0, 191, 179) !important;
```

Comment:
Is there a reason for this border color change?

Reply:
Not anymore. It's an unused class now so I've removed it. At one point we were thinking we wanted to make import stand out to be distinguishable as a very different process from the normal "Add layer" workflows.

Reply:
Gotcha. Can you remove this selector in the SASS file too then?

Reply:
Yep! Sorry, I meant that I'd removed the selector locally (not class). Will push up in just a little bit.